x-pack/test/security_solution_endpoint/apps/endpoint/artifact_entries_list.ts

```diff
@@ -114,39 +266,23 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
 
           // Checks if fleet artifact has been updated correctly
           await checkFleetArtifacts(
-            testData.fleetArtifact.type,
             testData.fleetArtifact.identifier,
             testData.fleetArtifact.getExpectedUpdatedtArtifactWhenCreate(),
-            testData.fleetArtifact.getExpectedUpdatedArtifactBodyWhenCreate()
+            testData.fleetArtifact.getExpectedUpdatedArtifactBodyWhenCreate(),
+            policyInfo
           );
         });
 
         it(`should be able to update an existing ${testData.title} entry`, async () => {
-          this.timeout(150_000);
-
-          // Opens edit flyout
-          await pageObjects.artifactEntriesList.clickCardActionMenu(testData.pagePrefix);
-          await testSubjects.click(`${testData.pagePrefix}-card-cardEditAction`);
-
-          for (const formAction of testData.update.formFields) {
-            if (formAction.type === 'click') {
-              await testSubjects.click(formAction.selector);
-            } else if (formAction.type === 'input') {
-              await testSubjects.setValue(formAction.selector, formAction.value || '');
-            } else if (formAction.type === 'clear') {
-              await (
-                await (await testSubjects.find(formAction.selector)).findByCssSelector('button')
-              ).click();
-            }
-          }
-
-          // Submit edit artifact form
-          await testSubjects.click(`${testData.pagePrefix}-flyout-submitButton`);
+          await createArtifact(testData);
+          await updateArtifact(testData, { policyId: policyInfo.packagePolicy.id });
 
           // Check edited artifact is in the list with new values (wait for list to be updated)
-          await retry.waitForWithTimeout('entry is updated in list', 10000, async () => {
+          await retry.waitForWithTimeout('entry is updated in list', 100000, async () => {
```

Comment:
Why sooooo long? :P

Reply:
This was a left over during development, will revert that. Thanks!

Reply:
https://github.com/elastic/kibana/pull/158061/commits/49e59753db9f8a16f5fdf3c020d6f6b7cbf35913